### PR TITLE
fixes #122  add search module with search functionality

### LINKED ISSUE #122: add search module with search functionality

### PR DESCRIPTION
**Pull request by tommysullivan**

### CODE REVIEW

**Comment by bob-bins** [comment]
i lose focus after every character input

File: src/components/layouts/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+export const SearchBar = (props) => {
+    const { searchBarValue } = props;
+    console.log({searchBarValue})
+
+    const getSuggestions = (searchBarValue) => {
+        const inputValue = searchBarValue.trim().toLowerCase();
+        const inputLength = inputValue.length;
+
+        return inputLength === 0 ? [] : organizations.filter(org =>
+            org.name.toLowerCase().slice(0, inputLength) === inputValue
```
Comment (bob-bins) [inline src/components/layouts/SearchBar.tsx]: can we instead filter by the org name containing the search term instead of starting with it?

Comment (PatrickMetsch) [reply src/components/layouts/SearchBar.tsx]: This is a work in progress PR. Sorry for not specifying. I'm just including pieces as I come to understand them. Just had a grand old time with the css.

I will definitely look in to figuring out how to search the organizations that /contain/ the search term, not just begin with it.

Comment (PatrickMetsch) [reply src/components/layouts/SearchBar.tsx]: Yeah, when the app refreshes it seems to pull autofocus to the Facebook module. I'm trying to figure out that bug.

Also, sorry for the late replies. Apparently when i set up Github to notify me, it didn't work out.

Comment (PatrickMetsch) [reply src/components/layouts/SearchBar.tsx]: test

**Comment by PatrickMetsch** [comment]
test

**Comment by PatrickMetsch** [comment]
Search bar now checks the entire organization title for the search term.

**Comment by PatrickMetsch** [comment]
Apparently, when the `component` prop of a `<Route />` calls React.createElement each time the app is rerendered, whereas using the `render` prop simply applies changes. Editing my route to use `render` fixed the loss of focus issue.

**Comment by bob-bins** [comment]
lookin good so far!

**Pull request merged**